pytradfri/device.py

```diff
@@ -215,8 +219,13 @@ def set_xy_color(self, color_x, color_y, *, index=0):
             ATTR_LIGHT_COLOR_Y: color_y
         }, index=index)
 
-    def set_kelvin_color(self, kelvins, *, index=0):
-        return self.set_values(kelvin_to_xyY(kelvins), index=index)
+    def set_hsb(self, hue, saturation, brightness, *, index=0):
+        """Set HSB color settings of the light."""
+        return self.set_values({
+            ATTR_LIGHT_COLOR_SATURATION: hue,
+            ATTR_LIGHT_COLOR_HUE: saturation,
+            ATTR_LIGHT_DIMMER: brightness
```

Comment:
@ggravlingen  Is it possible to support transition time here as well? I tried setting the transition time, and transitioning between colors works just fine :)

Reply:
@jurriaan I was planning on merging this now. Can you raise an issue please and I'll address it separately?

Reply:
That's okay, will do! :)